website/src/utils/yamlDataUtils/rename.js

```diff
@@ -78,11 +78,11 @@ const renameDBConnEvents = (path, oldValue, newValue, yamlData) => {
     // DB Connection
     const connSteps = ['core_lib', 'connections', path.at(-2)]
     const connTarget = getValueAtPath(data, connSteps)
-    connTarget['host'] = '${oc.env:' + newHost + '}'
-    connTarget['port'] = '${oc.env:' + newPort + '}'
-    connTarget['password'] = '${oc.env:' + newPassword + '}'
-    connTarget['username'] = '${oc.env:' + newUser + '}'
-    connTarget['file'] = '${oc.env:' + newDB + '}'
+    connTarget['url']['host'] = '${oc.env:' + newHost + '}'
```

Comment:
how come this is changed?

Reply:
it was a bug, the values were not in url in the connection